dashboards/k8s-storage-resources-glusterfs-pv.libsonnet

```diff
@@ -0,0 +1,291 @@
+          statPanelArgs +
+          g.statPanel('gluster:volume_capacity_used_bytes_total:sum%s / gluster:volume_capacity_total_bytes:sum%s' % [queryLabels, queryLabels])
+        )
+        .addPanel(
+          g.panel('Available Capacity') +
+          statPanelArgs +
+          g.statPanel('gluster:volume_capacity_total_bytes:sum%s - gluster:volume_capacity_used_bytes_total:sum%s' % [queryLabels, queryLabels], 'decbytes')
+        )
+        .addPanel(
+          g.panel('IOPS') +
+          statPanelArgs +
+          g.statPanel('(sum (gluster_volume_profile_total_reads_interval%s) + sum(gluster_volume_profile_total_reads_interval%s))/max(gluster_volume_profile_duration_secs_interval%s)' % [queryLabels, queryLabels, queryLabels], 'none')
+        )
+        .addPanel(
+          g.panel('Capacity Utilization') +
+          graphPanelArgs +
+          g.queryPanel('gluster:volume_capacity_used_bytes_total:sum%s' % queryLabels, '{{$Volume}}') +
+          g.stack +
+          { yaxes: g.yaxes('decbytes') },
+        )
+        .addPanel(
+          g.panel('IOPS') +
+          {
+            span: 6,
+            legend+: { show: false },
+          } +
+          g.queryPanel('(sum (gluster_volume_profile_total_reads_interval%s) + sum(gluster_volume_profile_total_reads_interval%s))/max(gluster_volume_profile_duration_secs_interval%s)' % [queryLabels, queryLabels, queryLabels], '{{$Volume}}') +
+          g.stack
+        )
+      )
+      .addRow(
+        g.row('LVM Summary')
+        .addPanel(
+          g.panel('LVM Thin Pool Metadata %') +
+          g.queryPanel('100*(gluster_thinpool_metadata_used_bytes%s / gluster_thinpool_metadata_total_bytes%s)' % [queryLabels, queryLabels], '{{thinpool_name}}')
+        )
+        .addPanel(
+          g.panel('LVM Thin Pool Data Usage  %') +
+          g.queryPanel('100*(gluster_thinpool_data_used_bytes%s / gluster_thinpool_data_total_bytes%s)' % [queryLabels, queryLabels], '{{thinpool_name}}')
+        )
+      )
+      .addRow(
+        (g.row('Bricks (Volume Subcomponent) Summary') +
+         {
+           showTitle: false,
+           panels: self._panels,
+         })
+        .addPanel(
+          g.panel('Bricks-Total') +
+          brickStatPanelArgs +
+          g.statPanel('gluster_volume_brick_count%s' % queryLabels, 'none')
+        )
+        .addPanel(
+          g.panel('Bricks-Up') +
+          brickStatPanelArgs +
+          g.statPanel('count (gluster_brick_up%s == 1)' % queryLabels, 'none') +
+          {
+            thresholds: '0.1,0.9',
+            colors: [
+              '#FFFFFF',
+              '#FFFFFF',
+              '#92d400',
+            ],
+            colorValue: true,
+          }
+        )
+        .addPanel(
+          g.panel('Bricks-Down') +
+          brickStatPanelArgs +
+          g.statPanel('count(gluster_brick_up%s) - count (gluster_brick_up%s == 1)' % [queryLabels, queryLabels], 'none') +
+          {
+            thresholds: '0.1,0.9',
```

Comment:
if the value is equal to or greater than 1 then the color should be red.

Reply:
Yes correct. The ideas is 0 - white, 1(or greater) - red